cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/planner/ConnectorDag.java

```diff
@@ -127,9 +127,8 @@ private ConnectorDag(Collection<Connection> connections,
 
     /*
         Find nodes that have input from multiple sources and add them to the connectors set.
-        We can probably remove this part once we support multiple sources. Even though we don't support
-        multiple sources today, the fact that we support forks means we have to deal with the multi-input case
-        and break it down into separate phases. For example:
+        We can probably remove this part once we support multiple sources. The fact that we support forks
```

Comment:
I think we should able to remove this entire section (lines 128 - 178), since its only used to break up multiple sources into separate phases. Or is there some reason only part of it was removed?